src/pages/Products/components/FilterBox.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './FilterBox.scss';
+
+const FilterBox = props => {
```

Comment:
```suggestion
const FilterBox = () => {
```
props 사용하지 않는다면 비워주세요!